src/modules/rtpengine/rtpengine.c

```diff
@@ -1784,6 +1784,8 @@ static int parse_flags(struct ng_flags_parse *ng_flags, struct sip_msg *msg, enu
 			case 6:
 				if (str_eq(&key, "to-tag")) {
 					ng_flags->to = 1;
+					if (val.s && val.len > 0)
```

Comment:
If a value is given, you could simply leave `ng_flags->to` alone (as zero) and `goto generic`